recipes/mysql-connector-c/all/conanfile.py

```diff
@@ -81,6 +84,13 @@ def package(self):
             os.remove(f)
         tools.rmdir(os.path.join(self.package_folder, "docs"))
 
+    @property
+    def _require_stdcpp_library(self):
+        return self.settings.compiler in ("gcc",) and not self.options.shared
+
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
         self.cpp_info.bindirs = ['lib']
+        if not self.options.shared:
+            if self._require_stdcpp_library:
+                self.cpp_info.system_libs.append("stdc++")
```

Comment:
Isn't this incoherent with `del self.settings.compiler.libcxx` above ?

Reply:
Probably, the interface is plain c, but internally some c++ is used.

Reply:
I think you could del compiler.libcxx or not, depending of that option. It will affect package id, but exporting only libstdc++ and hiding it from pkg id sounds wrong. Also, what happens when libc++ is used, instead of libstdc++?

Reply:
@ericLemanissier @uilianries  Please check my latest patches.